apps/backend/src/db/migrations/20240924130637_add_status_to_notifications_table.ts

```diff
@@ -0,0 +1,24 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = "notifications";
+
+const ColumnName = {
+	STATUS: "status",
+} as const;
+
+function up(knex: Knex): Promise<void> {
+	return knex.schema.alterTable(TABLE_NAME, (table) => {
+		table
+			.enu(ColumnName.STATUS, ["unread", "read"])
```

Comment:
It looks like `.enu` is the same as `.enum` but last one sounds better
```suggestion
			.enum(ColumnName.STATUS, ["unread", "read"])
```